composer.json

```diff
@@ -26,11 +26,18 @@
   "require" : {
     "php" : ">=7.1",
     "oat-sa/tao-core" : ">=50.5.0||dev-develop",
-    "elasticsearch/elasticsearch": "~7.0"
+    "elasticsearch/elasticsearch": "~8.13"
```

Comment:
Hey @tikhanovichA This library is not used anymore. My recommendation is to: 

- Archive this repository
- Update README.md here https://github.com/oat-sa/extension-tao-advanced-search?tab=readme-ov-file#warning

You will see there is no mention to this library in the code anymore